contrib/local-test/Makefile

```diff
@@ -1,12 +1,17 @@
 DGRAPH_VERSION ?= local
 
 current_dir = $(shell pwd)
-
+local-dgraph-data: 
+	mkdir -p ~/local-dgraph-data
 up: ## Start the zero and alpha containers
 	DGRAPH_VERSION=$(DGRAPH_VERSION) docker-compose up -d && docker-compose logs --follow alpha zero keepalive
 
-up-with-lambda: ## Start the zero and alpha containers and a lambda container
+up-with-lambda: local-dgraph-data## Start the zero and alpha containers and a lambda container
 	DGRAPH_VERSION=$(DGRAPH_VERSION) docker-compose -f docker-compose.yml -f docker-compose-lambda.yml up -d && docker-compose logs --follow alpha zero lambda keepalive
+up-with-lambda-acl: local-dgraph-data ## Start the zero and alpha containers and a lambda container
+	mkdir -p ~/local-dgraph-data/acl
```

Comment:
this kind of command can run into permission issues, e.g. on my local machine I saw
```
joshua@joshua-local:~/Developer/dgraph/contrib/local-test$ make up-with-lambda-acl
mkdir -p ~/local-dgraph-data
mkdir -p ~/local-dgraph-data/acl
mkdir: cannot create directory ‘/home/joshua/local-dgraph-data/acl’: Permission denied
make: *** [Makefile:12: up-with-lambda-acl] Error 1
```
which was because the docker container initially created the local-dgraph-data folder, so the owner is root:
```
drwxr-xr-x  6 root   root   4.0K May 23 17:40 local-dgraph-data
```
I will leave another comment with one way around this